flink-java/src/main/java/org/apache/flink/api/java/DataSet.java

```diff
@@ -521,7 +521,7 @@ public long count() throws Exception {
 		}
 
 		return new ReduceOperator<>(this, new SelectByMinFunction(
-				(TupleTypeInfo) getType(), fields), Utils.getCallLocationName());
```

Comment:
Can you please revert these style changes as well? We try to avoid touching files without a real need. Thanks